Jenkinsfile

```diff
@@ -1,73 +1,18 @@
-@Library('socrata-pipeline-library')
+@Library('socrata-pipeline-library@6.0.0') _
```

Comment:
This major release includes my refactoring commit of the sbt logic, but you can probably safely bump up to `6.1.0` if you want.

Reply:
Updated